renderdoc/driver/vulkan/vk_core.h

```diff
@@ -237,6 +237,23 @@ struct VulkanDrawcallCallback
   // callbacks above for the first EID, then call this function for the others
   // to indicate that they are the same.
   virtual void AliasEvent(uint32_t primary, uint32_t alias) = 0;
+
+  // Returns true if vkCmdExecuteCommands should be called separately for every
+  // command buffer in pCommandBuffers.
+  virtual bool SplitSecondary() { return false; }
```

Comment:
I'd prefer to keep this as a pure virtual interface rather than having default implementations. Can you move these empty/default implementations into the other uses of `VulkanDrawcallCallback`?

Reply:
Sure. Done.